utilities/test_suite/rpp_test_suite_audio.h

```diff
@@ -189,7 +201,33 @@ void read_from_bin_file(Rpp32f *srcPtr, RpptDescPtr srcDescPtr, Rpp32s *srcDims,
     free(refInput);
 }
 
-void verify_output(Rpp32f *dstPtr, RpptDescPtr dstDescPtr, RpptImagePatchPtr dstDims, string testCase, string dst, string scriptPath)
+//replicate the last sample buffer for the remaining samples
+void replicate_last_sample_mel_filter_bank(Rpp32f *srcPtr, int numSamples, unsigned long sampleSize, int batchSize)
+{
+    if (batchSize <= numSamples)
+        return;
+
+    Rpp32f *lastSample = srcPtr + (numSamples - 1) * sampleSize;
+    for (int i = numSamples + 1; i < batchSize; i++)
+    {
+        Rpp32f *sample = srcPtr + i * sampleSize;
+        std::memcpy(sample, lastSample, sampleSize * sizeof(Rpp32f));
```

Comment:
should i be starting from numSamples instead of numSamples + 1?

Reply:
changed